static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -128,24 +148,9 @@ class GroupHeader extends Component<Props, State> {
     }
 
     return [];
-  }
-
-  tabClickAnalyticsEvent(tab: Tab) {
-    const {organization, group, project} = this.props;
-    trackAdvancedAnalyticsEvent('issue_details.tab_changed', {
-      organization,
-      group_id: parseInt(group.id, 10),
-      issue_category: group.issueCategory,
-      project_id: parseInt(project.id, 10),
-      tab,
-    });
-  }
-
-  getErrorIssueTabs() {
-    const {baseUrl, currentTab, project, organization, group, location, replaysCount} =
-      this.props;
-    const disabledTabs = this.getDisabledTabs();
+  }, [organization, groupReprocessingStatus]);
 
+  const errrorIussueTabs = useMemo(() => {
```

Comment:
Seems like this could be its own component instead of a useMemo

Reply:
It's separate bc in the future we'll delete the performance only ones I think and it will just be this